### DISCUSSION TITLE
It would be nice to have autosave and a global environment

### DISCUSSION THREAD

**Discussion by insureyang** [usebruno/bruno#1341] [Ideas]
It would be nice to have autosave and a global environment～

**Comment by ysk3a** [comment]
I'd really like the globals environment feature.
In my case, I have more than one collections which uses globals environment variables, and a couple collections that sets the globals in postman, so if that can be a feature added to bruno, i could probably be able to completely leave postman unless required for special cases.

I think these are also related 
https://github.com/usebruno/bruno/discussions/1096 particularily for the `pm.globals.set(...)` and maybe https://github.com/usebruno/bruno/issues/691

some screenshots/sites for postman comparison:
https://www.postman.com/postman/workspace/postman-answers/globals
https://learning.postman.com/docs/sending-requests/variables/#variable-scopes
https://www.geeksforgeeks.org/how-to-create-and-use-global-variables-in-postman/ (in step 4)
